.eslintrc.json

```diff
@@ -1,14 +1,10 @@
 {
-  "root": true,
-  "extends": ["next/core-web-vitals"],
-  "plugins": ["testing-library"],
-  "overrides": [
-    {
-      "files": [
-        "**/__tests__/**/*.[jt]s?(x)",
-        "**/?(*.)+(spec|test).[jt]s?(x)"
-      ],
-      "extends": ["plugin:testing-library/react"]
-    }
-  ]
+   "root": true,
+   "extends": ["next", "prettier", "eslint:recommended", "plugin:@next/next/recommended"],
```

Comment:
best of eslint rules are ; "prettier", "eslint:recommended", "plugin:@next/next/recommended"